cli/src/main.rs

```diff
@@ -236,7 +236,7 @@ fn main() -> anyhow::Result<()> {
 
     #[cfg(feature = "x86-unicorn")]
     {
-        let mut trace_points = args.trace_points.unwrap();
+        let mut trace_points = args.trace_points.unwrap_or_default();
```

Comment:
I'm a little surprised this came up, maybe a leftover?  I fear I might have broken unicorn support at some point, I got it working to try to debug something and it ended up not helping so I didn't end up caring about it too much.

qemu's x86 emulator might be faster than retrowin32 though so I wonder if it's worth keeping around for performance reasons, hrmm...

Reply:
I originally was trying to use `x86-unicorn` before noticing that it was marked as "probably not working anymore", so I switched over to `x86-emu`, but left this fix in for good measure.

I think having a fast JIT x86 backend will eventually be useful, given that we want our compilers to execute as fast as possible. Maybe [FEX](https://github.com/FEX-Emu/FEX) will come in handy for x86 on arm64, it's much closer to Rosetta performance. qemu-user (and by extension, unicorn) isn't the speediest overall.

Reply:
I guess if your choices are "qemu's awesome emulator" and "evmar's hobby emulator" choosing unicorn is smart.  When I was using it before it was very slow so I hadn't taken it seriously but I wonder if I was somehow using it wrong, I should revisit it...